src/main/webapp/index.css

```diff
@@ -0,0 +1,62 @@
+html {
+    --container-color: #AEDFDB;
+    --info-container-color: #75C9CC;
+    --link-hover-color: #111111;
+    --link-color: white;
+    --list-color: #333333;
+}
+
+#home-page {
+    display: flex;
+    flex-direction: row;
+    height: 100vh;
+    padding: 0 !important;
+}
+
+#content-container {
+    background: var(--container-color);
+    flex-grow: 1;
+}
+
+#canvas {
+    display: block;
+    margin: auto;
+}
+
+canvas {
+    image-rendering: optimizeSpeed;
+    image-rendering: -moz-crisp-edges;
+    image-rendering: -webkit-optimize-contrast;
+    image-rendering: -o-crisp-edges;
+    image-rendering: crisp-edges;
+    -ms-interpolation-mode: nearest-neighbor;
+}
+
+h1 {
+    text-align: center;
+    padding: 1em;
+}
+
+li {
+  float: left;
+}
+
+li a {
+  display: block;
+  color: var(--link-color) !important;
+  padding: 16px;
+  text-align: center;
+  text-decoration: none;
+}
+
+li a:hover {
+  background-color: var(--link-hover-color);
+}
+
+ul {
+  background-color: var(--list-color);
+  list-style-type: none;
+  margin: 0;
+  overflow: hidden;
+  padding: 0;
+}
```

Comment:
Might need to move these to style.css so we have the same styles across pages. No change needed now though.